crates/rustc_codegen_c_ast/tests/blessed/test_decl_var.out

```diff
@@ -0,0 +1 @@
+int32_t _42 = 42;
```

Comment:
Could you adjust the test setup to make sure there is always a trailing newline? Sometimes files without a line ending can mess with tooling.